go/ct/st/code.go

```diff
@@ -142,3 +142,19 @@ func (c *Code) Copy() []byte {
 func (c *Code) String() string {
 	return fmt.Sprintf("%x", c.code)
 }
+
+// HumanReadableString returns a string with the length of the code and the
+// Human readable form for the opcodes in range [start, start+length).
+// - If the slice to be printed overflows the existing code, the overlapping code is printed.
+// - If start exceeds the code length, the length of the code is printed.
+func (c *Code) HumanReadableString(start int, length int) string {
```

Comment:
nit: This function name misses a verb ...

Reply:
I wanted to call it To.... but since the to_string function is called just string in go, I tried to stay idiomatic.

Reply:
I think the "String" function is called "String" because it produces a value of type "string".

Not sure about the idiomatic way in Go in this regard. I guess we are free to chose how we like it. Up to you.

Reply:
Renamed to ToXXXXXX

- c++ calls these functions to_string
- rust uses "to" prefix when a new instance is created, and "as" when a new type is returned but no new memory is allocated.